src/frontend/utils/parseDataElements/parseDataElements.spec.ts

```diff
@@ -70,9 +70,11 @@ describe.only('utils/parseDataElements', () => {
       const policyElement = document.createElement('div');
       policyElement.id = 'policy'; // triggers the creation of a nested object
       policyElement.setAttribute('data-policy', JSON.stringify(policy));
+      policyElement.setAttribute('data-modelname', 'policy');
```

Comment:
This is pluralized for `videos` etc. but singular for `policy`? Unless it's just an example, in which case it could stay closer to actual use